### PR TITLE
Bancor v3 Methodology v1.0.0

### PR DESCRIPTION
**Pull request by pietydespises**
Bancor v3 Methodology v1.0.0

### CODE REVIEW

**Comment by 0xbe1** [comment]
@this-username-is-taken why is this closed?

File: subgraphs/bancor-v3/README.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+Let’s assume that there are 1,500,000 LINK in the LINK liquidity pool on Bancor V3 and the current price of LINK at the time of calculation is $8.
+
+_Pool Total Value Locked USD (LINK Liquidity Pool) = Number of tokens in liquidity pool * price of token USD = 1,500,000 * $8 = $12,000,000 USD_
+
+**Reward Tokens & Reward Token Emissions Amount**
+
+**BNT Pool Reward Token Emissions**
+
+In Bancor V3 there will be a maximum of 40,000,000 BNT that will be distributed over an infinite time period.
```
Comment (tiagofilipenunes) [inline subgraphs/bancor-v3/README.md:283]: Breakdown of total BNT rewards:
30M BNT will be distributed over an infinite time period to BNT LPs
10M BNT to be distributed to TKN LPs, not necessarily under an exponential decay emission model i.e. over an infinite time period, as we also have the flat/linear emissions model mentioned below

**Review by tiagofilipenunes — COMMENTED** [review]
Thank you! Added a couple comments that I hope you'll find helpful

File: subgraphs/bancor-v3/README.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
+# Bancor V3 Subgraph Methodology v1.0.0
+
+## Useful Links
+
+* Bancor V3: [https://try.bancor.network/](https://try.bancor.network/)
+* Bancor V3 Proposal Document: 
+* Bancor V3 Technical Docs: [Bancor V3 Technical Docs](https://docs.bancor.network/about-bancor-network/bancor-v)
+* Introducing Bancor V3: [Introducing Bancor 3](https://blog.bancor.network/introducing-bancor-3-962a3c601c25)
+* Bancor V3 Useful Resources: [Resources - Bancor V3 Technical Docs](https://docs.bancor.network/about-bancor-network/resources)
+* Bancor V3 Presentation by Mark Richardson (Head of Bancor Research):
```
[the remark this answers is not in the document]
Comment (pietydespises) [reply subgraphs/bancor-v3/README.md:10]: fixed, thanks

File: subgraphs/bancor-v3/README.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+
+**Protocol Controlled Value USD**
+
+Protocol Controlled Value is relevant for Bancor V3. 
+
+The formula for the protocol controlled value for Bancor V3 will be as follows:
+
+_Total Protocol Controlled Value USD = Amount of vBNT Burned * Latest price of BNT in USD_
+
+This is how the official Bancor dune analytics dashboard estimated the BNT Value Locked Forever.
```
[the remark this answers is not in the document]
Comment (pietydespises) [reply subgraphs/bancor-v3/README.md:240]: added this to the document, thanks

File: subgraphs/bancor-v3/README.md [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+The formula for the protocol controlled value for Bancor V3 will be as follows:
+
+_Total Protocol Controlled Value USD = Amount of vBNT Burned * Latest price of BNT in USD_
+
+This is how the official Bancor dune analytics dashboard estimated the BNT Value Locked Forever. 
+
+Reference links
+
+* Dune Analytics Dashboard: [https://dune.com/Bancor/bancor_1](https://dune.com/Bancor/bancor_1)
+* Dune Analytics BNT Value Locked Forever Query: [https://dune.com/queries/30032/73935](https://dune.com/queries/30032/73935)
```
[the remark this answers is not in the document]
Comment (pietydespises) [reply subgraphs/bancor-v3/README.md:245]: noted, thanks

File: subgraphs/bancor-v3/README.md (hunk 1 above)
Comment (pietydespises) [reply subgraphs/bancor-v3/README.md:283]: Appreciate this info, thank you.

**Comment by this-username-is-taken** [comment]
See https://github.com/messari/subgraphs/pull/292

**Pull request closed without merging**